function.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable linebreak-style */
```

Comment:
- [ ] kindly consider avoiding disabling linters because it enforces a set of coding standards and style guidelines, ensuring that the code follows a consistent style across the entire project. Consistent code makes it easier to read, understand, and maintain, especially when multiple developers are working on the same codebase.